src/MobileApps/MyDriving/MyDriving.UWP/project.json

```diff
@@ -1,7 +1,7 @@
 ﻿{
   "dependencies": {
     "Acr.UserDialogs": "4.3.4",
-    "HockeySDK.UWP": "3.1.1-build59744",
+    "HockeySDK.UWP": "4.0.0",
```

Comment:
Looks good for me. Were you planning to update the version of Android and IOS versions of HockeyApp too ?